build.sbt

```diff
@@ -252,3 +230,173 @@ fixCSS := {
   if (("npm run fix-dark" #&& "npm run fix-light" !) != 0)
     throw new Exception("Error in CSS fix")
 }
+
+val pushCond                 = "github.event_name == 'push'"
```

Comment:
These seems useful, Could you comment whene these are triggered? e.g. will it be triggered by scala-steward dependencies?

Reply:
I copied these out of the existing workflows.
https://github.com/gemini-hlsw/explore/blob/a366ec0a2be4df7a844c45ffb8da1313f258c489/.github/workflows/full.yml#L63